sdk/spring/azure-spring-boot/src/test/java/com/azure/spring/aad/webapi/AADResourceServerOboConfigurationTest.java

```diff
@@ -24,6 +26,17 @@
             "azure.activedirectory.client-id=fake-client-id",
             "azure.activedirectory.client-secret=fake-client-secret");
 
+    @Test
+    public void testWithoutAnyPropertiesSet() {
```

Comment:
Do we need test like `testWithoutRequiredPropertiesSet`?

Reply:
Good point. I will add that.